plugins/offboard/include/plugins/offboard/offboard.h

```diff
@@ -92,7 +92,18 @@ class Offboard : public PluginBase {
         float yawspeed_deg_s; /**< @brief Yaw angular rate in degrees/second (positive for
                                  clock-wise looking from above). */
     };
-
+    
+    /**
+     * @brief Type for Attitude rate commands in body coordinates (roll, pitch, yaw  angular
+     * rate and Thrust).
+     */
+    struct SetAttitude {
+        float roll_deg_s;
+        float pitch_deg_s;
+        float yaw_deg_s;
+        float thrust_value;
```

Comment:
We we'll need comments for every field here because it's the public header file. This is used for the auto-generated reference docs.